### PR TITLE
added the transformer_interval table + API endpoint

### PR DESCRIPTION
**Pull request by derins**
Transformer Interval Table implemented with the API endpoint.

### CODE REVIEW

File: web/web/api/v1/transformer.py [hunk 1]
```diff
@@ -1,123 +0,0 @@
...
-from marshmallow import ValidationError
-from sqlalchemy.exc import IntegrityError
-from sqlalchemy.orm.exc import NoResultFound
-from .response_wrapper import ApiResponseWrapper
-from web.models.transformer import Transformer, TransformerSchema
-
-transformer_api_bp = Blueprint('transformer_api_bp', __name__)
-
-
-@transformer_api_bp.route('/transformers', methods=['GET'])
```
Comment (avpeery) [inline web/web/api/v1/transformer.py:12]: Does there not also need to be a transformer api?

**Review by avpeery — APPROVED** [review]

File: web/web/models/transformer_interval.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    unresp_load = Column(db.Float, nullable=False)
+
+    start_time = Column(TIMESTAMP, nullable=False)
+
+    end_time = Column(TIMESTAMP, nullable=False)
+
+    # Methods
+    def __repr__(self):
+        return f'<transformerInterval transformer_interval_id={self.transformer_interval_id} transformer_id={self.transformer_id}>'
```
Comment (jongoncalves) [inline web/web/models/transformer_interval.py:41]: Capitalize `<TransformerInterval`

**Review by jongoncalves — APPROVED** [review]

**Comment by derins** [comment]
@jongoncalves making the above change wouldn't require creating a new migration file right?

File: web/web/models/transformer_interval.py (hunk 2 above)
Comment (jongoncalves) [reply web/web/models/transformer_interval.py:41]: > @jongoncalves making the above change wouldn't require creating a new migration file right?

nope

**Pull request merged**